### ISSUE TITLE
[Question] How to check value of model trainable parameters?

### ISSUE THREAD

**Issue by mohammad-saber** [deepchem/deepchem#2710]
## ❓ Questions & Help

Hi, thank you for reading my question. 

When I initialize a graph model, I would like to check its trainable parameters (like model weights in MLP section). Is there any way to achieve that?

Also, how can I set the weight initializing method when making  a model?

**Comment by rbharath** [comment]
Yes, I believe you can do this, but details will depend on Torch vs Keras. I don't think we have a good example offhand.

Some of the models allow for choice of weight initialization I think but details would be in the docs
